.ruby-version

```diff
@@ -1 +1 @@
-2.7.4
+3.1.4
```

Comment:
`3.2.2` (the latest version of the next minor release) requires more dependency versions tango so I'm sticking to the latest `3.1` release